GOVERNANCE-elections.md

```diff
@@ -0,0 +1,274 @@
+
+There are two ways to raise issues to the steering committee for decision:
+
+1. Emailing the Steering Committee at
+   [steering@project.org](TODO).
+   This is a private discussion list to which all members of the committee have
+   access.
+2. Open an issue on [the community repository](TODO) and indicate that you would like
+   attention from the steering committee.
+
+## Composition
+
+The steering committee has [TODO: Number] seats. These five seats are
+open to any project contributor. See
+[candidate eligibility](#candidate-eligibility) for a definition.
+
+Steering Committee members serve for 2-year terms, staggered in order to
+preserve continuity. Every year either [TODO:Number] or [TODO:Remainder]
+contributor seats are elected.
+
+## Election Procedure
+
+### Timeline
+
+Steering Committee elections are held annually. Six weeks or more before the
+election, the Steering Committee will appoint Election Officer(s) (see below).
+Four weeks or more before the election, the Election Officer(s) will issue a
+call for nominations, publish the list of voters, and open the call for
+exceptions. One week before the election the call for nominations and exceptions
+will be closed. The election will be open for voting not less than two weeks and
+not more than four. The results of the election will be announced within one
+week of closing the election. New Steering Committee members will take office on
+[TODO: Date] of each year.
+
+### Election Officer(s)
+
+Six weeks or more before the election, the Steering Committee will appoint
+between one and three Election Officer(s) to administer the election. Elections
+Officers will be community members in good standing who are eligible to
+vote, are not running for Steering in that election, who are not currently part
+of the Steering Committee and can make a public promise of impartiality. They
+will be responsible for:
+
+- Making all announcements associated with the election
+- Preparing and distributing electronic ballots
+- Judging exception requests
+- Assisting candidates in preparing and sharing statements
+- Tallying voting results according to the rules in this charter
+
+### Eligibility to Vote
+
+Anyone who has at least [TODO:Number] contributions in the last 12 months is
+eligible to vote in the Steering election. Contributions are defined as opening PRs,
+reviewing and commenting on PRs, opening and commenting on issues, writing
+design docs, commenting on design docs, helping people on slack, participating
+in working groups, and other efforts that help advance the project.
+
+[This dashboard](TODO: Devstats link)
+shows only GitHub based contributions and does not capture all the contributions
+we value. We expect this metric not to capture everyone who should be eligible
+to vote. If a community member has had significant contributions over the past
+year but is not captured in Devstats,
+they will be able to submit an exception form to the Elections Officer(s) who
```

Comment:
If there is a template for the exception form, let's add that.

Reply:
Oh, I could create one of those!  That doesn't belong in this charter, but it would be very easy to add it in misc. templates.

Reply:
Can you open an issue on this repo for that request?